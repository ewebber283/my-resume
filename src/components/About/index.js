import React from 'react';

function About() {
  return (
    <section>
      <h1 id="about">Who am I?</h1>
    </section>
  );
}

export default About;