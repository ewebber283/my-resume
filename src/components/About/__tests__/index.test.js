import React from 'react'
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '..';

//ensure no leftover memory data
afterEach(cleanup);

describe('About component', () => {
    //baseline test-(whah is being tested, callback() => )
    it('renders', () => {
        render(<About />);
    });

    it('matches snapshot DOM node structure', () => {
        const { asFragment } =render(<About />);
        expect(asFragment()).toMatchSnapshot();
    });


  })