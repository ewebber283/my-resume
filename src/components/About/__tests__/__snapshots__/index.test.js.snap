// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="about"
    >
      Who am I?
    </h1>
  </section>
</DocumentFragment>
`;
